modules/social_features/social_core/social_core.install

```diff
@@ -1316,7 +1316,24 @@ function social_core_update_8908() {
 function social_core_update_10101() {
   $modules = [
     'ultimate_cron',
-    'social_advanced_queue'
+    'social_advanced_queue',
   ];
   \Drupal::service('module_installer')->install($modules);
 }
+
+/**
+ * Add a OS favicon on the admin pages.
+ */
+function social_core_update_10102() {
+  $config_file = drupal_get_path('module', 'social_core') . '/config/install/gin.settings.yml';
```

Comment:
Hi @navneet0693 @agami4 this is not a good idea, using static files is necessary here.